content/en/docs/reference/access-authn-authz/validating-admission-policy.md

```diff
@@ -365,3 +365,24 @@ HTTP response code, are used in the HTTP response to the client.
 The currently supported reasons are: `Unauthorized`, `Forbidden`, `Invalid`, `RequestEntityTooLarge`.
 If not set, `StatusReasonInvalid` is used in the response to the client.
 
+### Matching requests: `matchConditions`
+
+You can define _match conditions_for ValidatingAdmissionPolicy if you need fine-grained request filtering. These
+conditions are useful if you find that match rules, `objectSelectors` and `namespaceSelectors` still
+doesn't provide the filtering you want. Match conditions are
+[CEL expressions](/docs/reference/using-api/cel/). All match conditions must evaluate to true for the
+resource to be evaluated.
+
+Here is an example illustrating a few different uses for match conditions:
+
+{{< codenew file="access/validating-admission-policy-match-conditions.yaml" >}}
+
+Match conditions have access to the same CEL variables as validation expressions.
+
+In the event of an error evaluating a match condition the resource is never evaluated. Whether to reject
```

Comment:
S/the resource is never evaluated/ the policy is not evaluated

Reply:
I thought we evaluate the resource not the policy... But I might be wrong. Updated

Reply:
Kinda fuzzy. "The resource is evaluated against the policy" or "The policy is evaluated for the request" both read right to me.

The biggest concern I had was that "the resource is never evaluated" implies that the resource is not evaluated against any policy, which sounds scary. Changing the phrasing makes it clear that the criteria gates the policy and that the failure behavior is scoped to only the results of  the one policy.

Reply:
It makes sense :)